spec/models/card_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Card, :type => :model do
+  card = Card.new(original_text: "house", translated_text: "дом", transcription: "[haʊs]")
+  
+  it 'will return false' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.